processor/src/it/java/com/linecorp/decaton/processor/TestUtils.java

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright 2020 LINE Corporation
+ *
+ * LINE Corporation licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+
+package com.linecorp.decaton.processor;
+
+import java.util.Properties;
+import java.util.concurrent.CountDownLatch;
+
+import org.apache.kafka.clients.producer.ProducerConfig;
+
+import com.google.protobuf.MessageLite;
+
+import com.linecorp.decaton.client.DecatonClient;
+import com.linecorp.decaton.processor.SubscriptionStateListener.State;
+import com.linecorp.decaton.processor.runtime.ProcessorSubscription;
+import com.linecorp.decaton.processor.runtime.RetryConfig;
+import com.linecorp.decaton.processor.runtime.SubscriptionBuilder;
+import com.linecorp.decaton.protobuf.ProtocolBuffersSerializer;
+
+public class TestUtils {
```

Comment:
I think this will be useful for any integration tests that might be implemented for different module in the future. How about moving this to testing module?

Reply:
Sounds make sense.